frame/nfts/src/impl_nonfungibles.rs

```diff
@@ -173,8 +174,40 @@ impl<T: Config<I>, I: 'static> Create<<T as SystemConfig>::AccountId, Collection
 			T::CollectionDeposit::get(),
 			Event::Created { collection, creator: who.clone(), owner: admin.clone() },
 		)?;
+
+		Self::set_next_collection_id(collection);
+
 		Ok(collection)
 	}
+
+	/// Create a collection of nonfungible items with `collection` Id to be owned by `who` and
+	/// managed by `admin`. Should be only used for applications that do not have an incremental
+	/// order for the collection IDs and is a replacement for the auto id creation.
+	///
+	///
+	/// SAFETY: This function can break the pallet if it is used in combination with the auto
+	/// increment functionality, as it can claim a value in the ID sequence.
```

Comment:
```suggestion

	/// Create a collection of non-fungible items with a `collection` ID to be owned by `who` and
	/// managed by `admin`. This is a replacement for the auto ID creation (handled by the `create_collection` function) and should only be used by applications that do not require an incremental order of collection IDs.
	///
	///
	/// SAFETY: This function will break the pallet if it is used in combination with automatic ID
	/// incrementation as it can claim a value that already exists.
```